docs/contributing.md

```diff
@@ -57,33 +57,6 @@ Use the [Discussions](https://github.com/DaveSkender/Stock.Indicators/discussion
 - We expect all unit tests to execute successfully and all Errors and Warning resolved before you submit your code.
 - Failed builds or unit testing will block acceptance of your Pull Request when submitting changes.
 
-### Performance benchmarking
```

Comment:
You over did the removals here.  We should still have this section with brief usage instructions for the most common commands, which were there; we just need to remove the blurbs with links to the "comprehensive guidance" page we removed.